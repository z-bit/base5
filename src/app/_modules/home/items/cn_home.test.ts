import { TestBed, ComponentFixture } from '@angular/core/testing';
import { NO_ERRORS_SCHEMA } from '@angular/core';
import { HomeComponent } from './cn_home.component';

// import { StoreModule, Store, combineReducers } from '@ngrx/store';
// import * as Auth from '../actions/auth';
// import * as fromAuth from '../reducers';


describe('AppComponent (app-root)', () => {
  let fixture: ComponentFixture<HomeComponent>;
  let instance: HomeComponent;
  
  beforeEach(() => {
    TestBed.configureTestingModule({
      declarations: [HomeComponent],
      schemas: [NO_ERRORS_SCHEMA],
    });
    
    fixture = TestBed.createComponent(HomeComponent);
    instance = fixture.componentInstance;
  });
  
  it('should compile', () => {
    fixture.detectChanges();
    
    /**
     * The login form is a presentational component, as it
     * only derives its state from inputs and communicates
     * externally through outputs. We can use snapshot
     * tests to validate the presentation state of this component
     * by changing its inputs and snapshotting the generated
     * HTML.
     *
     * We can also use this as a validation tool against changes
     * to the component's template against the currently stored
     * snapshot.
     */
    expect(fixture).toMatchSnapshot();
  });
});