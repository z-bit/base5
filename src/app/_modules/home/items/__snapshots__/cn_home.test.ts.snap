// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppComponent (app-root) should compile 1`] = `
<app-home>
  <h1>
    Home
  </h1>
</app-home>
`;
